dashboards-observability/.cypress/utils/event_constants.js

```diff
@@ -136,5 +175,26 @@ export const renderDataConfig = () => {
 export const renderLineChart = () => {
   landOnEventVisualizations();
   querySearch(TEST_QUERIES[5].query, TEST_QUERIES[5].dateRangeDOM);
-  cy.get('[data-test-subj="configPane__vizTypeSelector"] [data-test-subj="comboBoxInput"]').type('Line').type('{enter}');
+  cy.get('[data-test-subj="configPane__vizTypeSelector"] [data-test-subj="comboBoxInput"]')
+  .type('Line')
+  .type('{enter}');
+};
+
+export const renderDataConfigTreeMap = () => {
```

Comment:
this can be moved to event analytics  file as a common method.

Reply:
We add all the common methods in event_constants.js page.